packages/css/src/grid/grid-cell.scss

```diff
@@ -5,14 +5,20 @@
 
 @import "../../utils/breakpoint";
 
-@media screen and (width > $amsterdam-breakpoint) {
-  @for $i from 1 through 12 {
-    .amsterdam-grid-column-start-#{$i} {
-      grid-column-start: $i;
-    }
+.amsterdam-grid-cell:first-child {
+  grid-column-start: content;
+}
+
+.amsterdam-grid-cell.amsterdam-grid-cell-full-width {
+  grid-column: full;
```

Comment:
This is a shorthand for `grid-column-start: full-start; grid-column-end: full-end`.